### PR TITLE
Added Pixel Shader for Nomads and tweaked bloom values

### PR DESCRIPTION
**Pull request by ThomasHiatt**
Copied and renamed the old NormalMappedPS function to
NomadNormalMappedPS and updated all nomad techniques that were using the
old version to use the new version. The new version adjusts the color
and alpha values to give things a bit more glow while not desaturating
the color.

### CODE REVIEW

**Comment by CookieNoob** [comment]
uhm. Thats not what we wanted actually... You change the current shader instead of introducing a new shader to which we would switch the units to that should use it.

What it should instead do:
introduce a new technique NomadsUnit2 that uses the new pixelshader
leave the old function NomadUnit unchanged, but rename it to NomadsUnit1

**Comment by Exotic-Retard** [comment]
actually what cookie wrote is not quite right either xD
we want to keep a set of not renamed techniques temporarily so that when its updated in base faf it doesnt break nomads so we dont have to immediately release a patch

but we DO want to eventually rename all NomadSomething to NomadsSomething
on top of that we want 2 versions of NomadUnit - NomadsUnit and NomadsUnit2 using the normal and new pixel shader respectively.

**Comment by CookieNoob** [comment]
you could also use the new naming convention for the pixel shader: `NomadNormalMappedPS` -> `NomadsNormalMappedPS`

File: effects/mesh.fx [hunk 1]
```diff
@@ -7727,6 +7781,960 @@ technique ShieldNomadStealth_LowFidelity
...
+    // A pass afterwards must write the depth buffer
+    pass P0
+    {
+        RasterizerState( Rasterizer_Cull_None )
+        AlphaState( AlphaBlend_SrcAlpha_InvSrcAlpha_Write_RGBA )
+        ZEnable = true;
+        ZWriteEnable = false;
+
+        VertexShader = compile vs_1_1 NOMADBuildVS();
+        PixelShader = compile ps_2_a NomadBuildHologramPS(true);
```
Comment (CookieNoob) [inline effects/mesh.fx]: why is the nomads technique using the nomad (no s) shaders?

**Comment by aeoncleanse** [comment]
Or we could ensure that faf and nomads patches are pushed at the same time like sensible people, so this is totally not needed

**Comment by CookieNoob** [comment]
By leaving the not renamed shader techniques in the mesh file, we are independent from the FAF patch cycle for the time until we renamed all shaders in nomads.

Having a wrong name for the technique for a single unit that's on the field will cause a hard game crash. A smooth transition (adding new techniques, then rename them in the units, then remove the old techniques) is the best way to do it.

**Comment by aeoncleanse** [comment]
Would someone like to tell me why code dedicated to and used only by a mod is found in the primary game files?

**Comment by CookieNoob** [comment]
as you probably saw, this code was added a loooonnng time ago by ZeP as it was intended to implement nomads into the main game as soon as the closed beta phase was over. It wasn't directly merged to allow for a faster update and patch cycle during development.

There are more parts in the code that are only there to allow for nomads support. For example the movies folder in the ProgramData/FAForever including the updater was added to make the nomads loading screen work (it was added by ZeP but broke/never worked).

one of the many mentions of nomads being implemented in main FAF: http://direct.faforever.com/2013/07/patch-3626-next/#.WOYTvSf6xFQ

**Comment by aeoncleanse** [comment]
I'm gonna have to discuss which way to go about this in the next Dev conference. Personally I'd rather pull all Nomads code out of the base game, a mod is a mod.

**Comment by CookieNoob** [comment]
sure, dont forget to make a PR to add all of this code to nomads then.

**Comment by CookieNoob** [comment]
sure, dont forget to make a PR to add all of this code to nomads then.

It's probably unnecessary to remind you that FAF is also a mod, and some mod compatibility doesnt hurt. So if you remove the code, make sure nomads still works.

**Comment by aeoncleanse** [comment]
Technically FAF is a mod. Logistically FAF has no more been a mod than the core game for some years now. 

Removing dedicated code not ever accessed by 99% of the userbase is not removing compatibility. Removing compatibility to block Nomads working would be removing compatibility.

**Comment by Exotic-Retard** [comment]
removing shader changes from here would block nomads working. until you put a pr with the shader changes in nomads working from their own repo, that would be breaking compatibility. and yes i have already tried putting them in there but it didnt work, so its your turn.

same goes for all the other parts of nomads in faf repo. theyre not there just because we felt like it.

**Comment by aeoncleanse** [comment]
They can be modded, you have a full sim hook. Just shadow the current repo if you must, but this code doesnt belong here. Im not suggesting a patch removing this be pushed before nomads is ready to or already has adapted

**Comment by aeoncleanse** [comment]
I will merge this for now. I have also created a new issue: #1906

**Pull request merged**